planet/sync/__init__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Planet Labs, Inc.
+# Copyright 2022 Planet Labs PBC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from .data import DataAPI
+from .orders import OrdersAPI
+from .subscriptions import SubscriptionsAPI
+from .client import Planet
+
+__all__ = ['DataAPI', 'OrdersAPI', 'Planet', 'SubscriptionsAPI']
+
+# Organize client classes by their module name to allow lookup.
```

Comment:
I don't think we need/want this ... this is a relic of a bizarre pattern that I believe came from dealing w/ circular deps